tests/test_guard_body.py

```diff
@@ -99,3 +99,171 @@ def post(req):
 
         when(form=given | dict(bar=0))
         assert status == '400 bar: Zero Not Allowed'
+
+
+def test_bodyguard_json_input(app, Given):
```

Comment:
remove the `_input` suffix from the function name(s).

Reply:
Done